guides/micronaut-rabbitmq-rpc/bookinventory/groovy/src/main/groovy/example/micronaut/BookInventory.groovy

```diff
@@ -0,0 +1,21 @@
+package example.micronaut
+
+import groovy.transform.CompileStatic
+import groovy.transform.EqualsAndHashCode
+import io.micronaut.core.annotation.Creator
+import io.micronaut.core.annotation.Introspected
+
+@Introspected
+@CompileStatic
+@EqualsAndHashCode
+class BookInventory {
+
+    final String isbn
+    final Integer stock
+
+    @Creator
```

Comment:
Is `@Creator` necessary? If you want to include it, we should educate the user about it with a callout.